### PR TITLE
Zendesk

### PR DESCRIPTION
**Pull request by sebgoa**
first zendesk flow

### CODE REVIEW

File: bridges/zendesk/cron-zendesk.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      spec:
+        email: ""
+        subdomain: ""
+        subject: ""
+        token:
+          secretKeyRef:
+            key: token
+            name: ""
+    options:
+      enableResync: false
```
Comment (JeffNaef-Triggermesh) [inline bridges/zendesk/cron-zendesk.yaml:57]: 👀 havent seen `enableResync` before! ill have to check this out.

Comment (antoineco) [reply bridges/zendesk/cron-zendesk.yaml:57]: This field is automatically added in Bridges created via the UI. @odacremolbap should be able to clarify the use, but my assumption is that it's implicitly On by default.

Comment (odacremolbap) [reply bridges/zendesk/cron-zendesk.yaml:57]: I'll work on this to clarify https://github.com/triggermesh/triggerflow/issues/115

but for the record, that flag does not work any longer.

There are policies regarding resync and deletion that can be applied to bridges (as a default to all components) and individually to components.

- Bridges: 
https://github.com/triggermesh/triggerflow/blob/2cda0fbd0d3b7b93c9ecc149bcc5be44fc3d8777/pkg/apis/flow/v1alpha1/bridge_types.go#L95-L100

- Components:
https://github.com/triggermesh/triggerflow/blob/2cda0fbd0d3b7b93c9ecc149bcc5be44fc3d8777/pkg/apis/flow/v1alpha1/bridge_types.go#L113-L137

It defaults to the `bridge` value that means
- a component will resync its contents if the object was created by a bridge and the bridge being resynced is the _dominant_ bridge. If the component already existed and was not created by a bridge (for example the default broker) wont get its contents resynced.
- on bridge deletion a component will be deleted if it was created by a bridge, and the bridge being deleted is the last bridge referencing the component.

Comment (odacremolbap) [reply bridges/zendesk/cron-zendesk.yaml:57]: TLDR;

remove any flag, default use should be good and we shouldn't be using any other flavour until UI supports it.

**Comment by sebgoa** [comment]
@JeffNaef-Triggermesh what's the status on this?

**Comment by JeffNaef-Triggermesh** [comment]
@sebgoa I dont know sir. I didn't know that this was mine. I saw that you had made it and gave it some review last week but that was it. I will fix what snr Pablo commented on and we can get it merged 😁

**Comment by JeffNaef-Triggermesh** [comment]
ok. there we go. tested, working, and ready to merge :)

**Comment by sebgoa** [comment]
Right, I submitted this PR :(

**Pull request merged**